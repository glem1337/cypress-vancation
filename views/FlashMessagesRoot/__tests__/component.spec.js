import React from 'react';
import { shallow } from 'enzyme';

import fakeIntl from 'utils/testHelpers/fakeIntl';

import MessagesRootComponent from '../component';

describe('MessagesRootComponent component tests', () => {
  const props = {
    messages: {
      1: { id: '1', duration: 3, messageTitle: {}, messageType: 'success' },
    },
    intl: fakeIntl,
    hideMessage: jest.fn(),
  };

  const component = shallow(<MessagesRootComponent {...props} />);

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
