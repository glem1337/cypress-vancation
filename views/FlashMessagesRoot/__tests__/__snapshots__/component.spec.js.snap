// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessagesRootComponent component tests matches snapshot 1`] = `
<Fragment>
  <Message
    duration={3}
    hideMessage={[MockFunction]}
    id="1"
    intl={
      Object {
        "formatMessage": [Function],
      }
    }
    key="1"
    messageTitle={Object {}}
    messageType="success"
  />
</Fragment>
`;
